pkg/volume/testing/testing.go

```diff
@@ -62,15 +63,16 @@ const (
 
 // fakeVolumeHost is useful for testing volume plugins.
 type fakeVolumeHost struct {
-	rootDir    string
-	kubeClient clientset.Interface
-	pluginMgr  VolumePluginMgr
-	cloud      cloudprovider.Interface
-	mounter    mount.Interface
-	exec       mount.Exec
-	nodeLabels map[string]string
-	nodeName   string
-	subpather  subpath.Interface
+	rootDir         string
+	kubeClient      clientset.Interface
+	pluginMgr       VolumePluginMgr
+	cloud           cloudprovider.Interface
+	mounter         mount.Interface
+	exec            mount.Exec
+	nodeLabels      map[string]string
+	nodeName        string
+	subpather       subpath.Interface
+	csiDriverLister *storagelisters.CSIDriverLister
```

Comment:
I don't see any tests actually using this (maybe i'm not looking hard enough). If there aren't do we really need to add this to the struct and have all the interface functions?

Reply:
It is used in unit tests under pkg/volume/csi.  Without this unit tests will fail.